<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Game</class>
 <widget class="QWidget" name="Form_Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="BACKGROUND">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="Widget_PlayerC" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Widget_PlayerC{
	border: 1px solid black;
	border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_pfp_playerC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 30px;
	background-color: transparent;
}
QPushButton:hover{
	border: 1px solid red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pfp_playerC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cardNum_playerC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_cardNum_playerC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt;</string>
    </property>
    <property name="text">
     <string>剩余：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_landlord_playerC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lbl_pfp_playerC</zorder>
   <zorder>btn_pfp_playerC</zorder>
   <zorder>lbl_cardNum_playerC</zorder>
   <zorder>lbl_title_cardNum_playerC</zorder>
   <zorder>lbl_landlord_playerC</zorder>
  </widget>
  <widget class="QWidget" name="Widget_PlayerB" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Widget_PlayerB{
	border: 1px solid black;
	border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_pfp_playerB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 30px;
	background-color: transparent;
}
QPushButton:hover{
	border: 1px solid red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pfp_playerB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cardNum_playerB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_cardNum_playerB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt;</string>
    </property>
    <property name="text">
     <string>剩余：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_landlord_playerB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lbl_pfp_playerB</zorder>
   <zorder>lbl_cardNum_playerB</zorder>
   <zorder>lbl_title_cardNum_playerB</zorder>
   <zorder>lbl_landlord_playerB</zorder>
   <zorder>btn_pfp_playerB</zorder>
  </widget>
  <widget class="QWidget" name="Widget_CardLbls" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>560</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>110</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="HSpacer_left_CardLbls">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>135</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="HSpacer_right_CardLbls">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>135</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_time_playerB">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(50,50,50);
border-radius: 20px;
font: 30pt;
color: rgba(255,0,0,200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_playerC">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(50,50,50);
border-radius: 20px;
font: 30pt;
color: rgba(255,0,0,200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="Widget_PlayerA" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>781</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QLabel" name="lbl_landlord_playerA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="Widget_PlayerInfo" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="btn_pfp_playerA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 30px;
	background-color: transparent;
}
QPushButton:hover{
	border: 1px solid red;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pfp_playerA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_username_playerA">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>15</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>lbl_pfp_playerA</zorder>
    <zorder>btn_pfp_playerA</zorder>
    <zorder>lbl_username_playerA</zorder>
   </widget>
   <widget class="QLabel" name="lbl_time_playerA">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(50,50,50);
border-radius: 20px;
font: 30pt;
color: rgba(255,0,0,200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="Widget_CardBtns" native="true">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>90</y>
      <width>695</width>
      <height>145</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>695</width>
      <height>145</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>695</width>
      <height>145</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#Widget_CardBtns{
	border: 1px solid balck;
	border-radius: 10px;
}

QPushButton{
	border: 1px solid grey;
	border-radius: 3px;
	background-color: rgb(255, 255, 250);
	font: 18pt;
}
QPushButton:hover{
	background-color: rgb(255, 255, 240);
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="HSpacer_left_CardBtns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>135</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="HSpacer_right_CardBtns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>135</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Widget_BTN_PlayCard" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_pass_playCard">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>5</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>不出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_playCard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>出牌</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Widget_BTN_Landlord" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_pass_landlord">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>5</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>不做</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_become">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>做地主</string>
     </property>
    </widget>
   </widget>
   <zorder>lbl_landlord_playerA</zorder>
   <zorder>Widget_PlayerInfo</zorder>
   <zorder>lbl_time_playerA</zorder>
   <zorder>Widget_CardBtns</zorder>
   <zorder>Widget_BTN_Landlord</zorder>
   <zorder>Widget_BTN_PlayCard</zorder>
  </widget>
  <zorder>BACKGROUND</zorder>
  <zorder>Widget_PlayerC</zorder>
  <zorder>Widget_PlayerB</zorder>
  <zorder>Widget_CardLbls</zorder>
  <zorder>lbl_time_playerC</zorder>
  <zorder>lbl_time_playerB</zorder>
  <zorder>Widget_PlayerA</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
